### ISSUE TITLE
Miner does not use multiple processors[issue name]

### ISSUE THREAD

**Issue by starnodes** [iron-fish/ironfish#957]
### What happened?

Hello!
I updated to version 0.1.21 and noticed that my 2 Intel(R) Xeon(R) CPU E5-4660 v3 processors are only 50% loaded. In the launch settings, the option start -t -1 is specified.
My Graffiti is "ithype"

### Version

0.1.21

### Relevant log output

```shell
Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   46 bits physical, 48 bits virtual
CPU(s):                          56
On-line CPU(s) list:             0-55
Thread(s) per core:              2
Core(s) per socket:              14
Socket(s):                       2
NUMA node(s):                    2
Vendor ID:                       GenuineIntel
CPU family:                      6
Model:                           63
Model name:                      Intel(R) Xeon(R) CPU E5-4660 v3 @ 2.10GHz
Stepping:                        2
CPU MHz:                         2893.078
CPU max MHz:                     2900.0000
CPU min MHz:                     1200.0000
BogoMIPS:                        4189.97
Virtualization:                  VT-x
L1d cache:                       896 KiB
L1i cache:                       896 KiB
L2 cache:                        7 MiB
L3 cache:                        70 MiB
NUMA node0 CPU(s):               0-13,28-41
NUMA node1 CPU(s):               14-27,42-55
```

**Comment by AmberKiso** [comment]
Take a look at this comment thread in the Discord server: https://discord.com/channels/771503434028941353/828803908067262484/948444206861541386 (#mining)
People mining seriously have experimented with the batch size parameter to get better performance out of their rigs.
Try `yarn start:once config:set minerBatchSize 50000` and experiment with that value to see what hash rate you can generate.
You can use `yarn start:once config:show` to see what the current value of the `minerBatchSize` parameter is.

**Comment by deekerno** [comment]
This should have been addressed through the mining refactor released in [v0.1.26](https://github.com/iron-fish/ironfish/tree/v0.1.26). If you continue to encounter this issue, feel free to re-open this issue.
